### PR TITLE
Fixes #33931 - update kickstart custom template

### PR DESCRIPTION
**Pull request by eb4x**
Just some minor cleanups to the kickstart_custom template.

### CODE REVIEW

File: app/views/unattended/partition_tables_templates/kickstart_custom.erb [hunk 1]
```diff
@@ -19,35 +19,22 @@ clearpart --all --initlabel
   fstype = host_param('part_fstype') || 'xfs'
 -%>
 
-<% if @host.pxe_loader.include?('UEFI') -%>
-
+<% if @host.pxe_loader.include?('UEFI') or @host.pxe_loader.include?('BIOS') -%>
```
Comment (ekohl) [inline app/views/unattended/partition_tables_templates/kickstart_custom.erb]: In Ruby `||` is preferred
```suggestion
<% if @host.pxe_loader.include?('UEFI') || @host.pxe_loader.include?('BIOS') -%>
```

File: app/views/unattended/partition_tables_templates/kickstart_custom.erb [hunk 2]
```diff
@@ -19,35 +19,22 @@ clearpart --all --initlabel
...
+  part /boot --fstype="<%= fstype %>" --ondisk=<%= dev %> --size=1024
   <% else -%>
-    part /boot --fstype="ext4" --ondisk=<%= dev %> --size=1024
+  part /boot --fstype="ext4" --ondisk=<%= dev %> --size=1024
   <% end -%>
   part pv.01 --fstype="lvmpv" --ondisk=<%= dev %> --size=1024 --grow
-  volgroup vg_sda --pesize=4096 pv.01
-  logvol swap --fstype="swap" --size=<%= swap %> --name=lv_swap --vgname=vg_sda
-  logvol / --fstype="<%= fstype %>" --percent=<%= root %> --name=lv_root --vgname=vg_sda
+  volgroup vg_<%= dev %> --pesize=4096 pv.01
```
Comment (ekohl) [inline app/views/unattended/partition_tables_templates/kickstart_custom.erb]: Would it make sense to define an additional variable for the volume group name?

**Review by ekohl — COMMENTED** [review]
Some minor notes inline, but overall this looks correct and a nice cleanup.

File: app/views/unattended/partition_tables_templates/kickstart_custom.erb (hunk 1 above)
Comment (eb4x) [reply app/views/unattended/partition_tables_templates/kickstart_custom.erb]: Thanks, I've updated as suggested :+1:

File: app/views/unattended/partition_tables_templates/kickstart_custom.erb (hunk 2 above)
Comment (eb4x) [reply app/views/unattended/partition_tables_templates/kickstart_custom.erb]: Yeah, I actually wanted to do that, but then you'd probably want to customize lv_swap lv_root and lv_home also.

I'm not up to speed on ruby templates, in jinja I'd do something like `{{ vgname | default('vg_' ~ dev) }}` If you can give me a hint on how to structure it I can create a issue/feature-request and implement it.

Comment (ekohl) [reply app/views/unattended/partition_tables_templates/kickstart_custom.erb]: Between line 14 and 20 those are defined. We use `|| 'default'`. In this case you would use `|| "vg_#{dev}"` to perform the string interpolation. Note string interpolation requires `"` and `'` doesn't interpolate. Also be sure it's below the definition of `dev`.

Comment (eb4x) [reply app/views/unattended/partition_tables_templates/kickstart_custom.erb]: Thanks :) Took a stab at customizing the names.

**Comment by ekohl** [comment]
ok to test

**Review by lzap — DISMISSED** [review]
Please squash into a single commit.

**Review by lzap — APPROVED** [review]
Ok since this is not the default template we associate OS with, I am good. Not tested. Thank you!

**Pull request merged**